tox.ini

```diff
@@ -1,9 +1,20 @@
 [tox]
 envlist =
-    py27, pypy, py34, py35, py36
+    py27, pypy, py34, py35, py36, py37, coverage
 
 [testenv]
 commands =
     zope-testrunner --test-path=src []
 deps =
     .[test]
+
+[testenv:coverage]
+usedevelop = true
+basepython =
+    python3.6
+commands =
+    coverage run -m zope.testrunner --test-path=src
+    coverage report --fail-under=100
+deps =
+    .[test]
```

Comment:
I'm mildly wondering about `extas = test` instead of `deps = .[test]`, but meh.

Reply:
TIL about `extras` (thanks). So I was thinking of making my new thing 'usedevelop = true' and 'extras = test'. That avoids the sdist step and is faster...but if there are MANIFEST.in problems they might not show up.

Reply:
I'm more comfortable not using `usedevelop` by default, because it seems safer to test installed code to catch extra problems.  Developers can do `tox --develop` if they wish to get faster feedback loops.

(The one exception is when lack of `usedevelop` makes coverage painful.)